MetamodelAlgemeen.md

```diff
@@ -1617,18 +1617,51 @@ In essentie zijn er vier componenten die een meting of een waarneming beschrijve
 
 1. het _onderwerp_ (wat wordt er gemeten)
 1. de _waarde_ (de waarde die gemeten is)
-1. het _datatype_ van die waarde (kwalitatief (`Characterstring`), kwantitatief (`Integer` of `Real`))
+1. het _datatype_ van die waarde (kwalitatief:`Characterstring`, kwantitatief:`Integer` of `Real`)
```

Comment:
Is dit niet te limitatief? Waarom kan Decimal, Boolean, Date, DateTime, Year, Day, Month geen meting of waarneming beschrijven?

Hier wordt denk ik bedoeld dat er bepaalde metingen of waarnemingen zijn die nog een eenheid kunnen hebben om de waarde te kwalificeren.
Dat zou dan scherper geformuleerd moeten worden en dan mist in ieder geval `Decimal` nog.

Reply:
Zelf las ik het als voorbeelden; niet als uitputtende lijst van mogelijke datatypen. Dit was al een aanscherping van de originele tekst. Dit kan (en moet misschien ook) scherper. 

Ik vraag mij alleen af op welke manier en hoeveel het MIM daarover moet zeggen. Dit is een thema op zichzelf. Stel je eenheid is `dag`. Dat is geen SI-eenheid, maar wel een toegestane niet-SI-eenheid. De _unit expression_ is: `d`. Maar wat is de **waarde** van deze eenheid: een getal (`[0-6]`, of `[1 -7]`) of tekst (de naam van de dag), of een waarde uit een waardelijst? In welke taal? Wat is de schrijfwijze (hoofd-/kleine letters) En als het Engels is, moet er dan een Nederlandse vertaling komen voor MIM?

Reply:
Decimal moet er zeker bij. Bij de andere kan ik me geen voorstelling maken. Tijd zit in eenheid verwerkt. Boolean als kwantitatieve waarde?

Reply:
Op basis van bovenstaande zijn `Boolean` en `Decimal` nu toegevoegd.